Sources/Controllers/ALKConversationViewController.swift

```diff
@@ -796,11 +804,12 @@ open class ALKConversationViewController: ALKBaseViewController, Localizable {
     // swiftlint:disable:next cyclomatic_complexity function_body_length
     private func prepareChatBar() {
         // Update ChatBar's top view which contains send button and the text view.
-        chatBar.grayView.backgroundColor = configuration.backgroundColor
+        chatBar.grayView.backgroundColor = UIColor.dynamicColor(light: configuration.backgroundColor, dark: UIColor(netHex: 0x1C1C1C))
 
         // Update background view's color which contains all the attachment options.
-        chatBar.bottomBackgroundColor = configuration.chatBarAttachmentViewBackgroundColor
+        chatBar.bottomBackgroundColor = UIColor.dynamicColor(light: configuration.chatBarAttachmentViewBackgroundColor, dark: UIColor(netHex: 0x313131))
```

Comment:
0x1C1C1C & 0x313131 these also used different places & hardcoded in everyplace. instead we can make it constant  single place & use it in all places

Reply:
done